canonical-websites/www.ubuntu.com
# Hygiene: Synergise naming of v1 Sass partials

## Done

Give consistent naming to partial names and mixins introduced by myself and @deadlight respectively.

## QA

- Pull code
- Sanity check code
- Run `node-sass --include-path node_modules static/sass --output static/css`
- Ensure no errors

## Issue / Card
